dynamics_class: "dubins_car"
safe_set:
  lo: 
    - -5.0
    - -5.0
    - -3.1416
  hi: 
    - 5.0
    - 5.0
    - 3.1416

obstacles: []

state_domain:
  lo: 
    - -5.0
    - -5.0
    - -3.1416
  hi: 
    - 10.0
    - 10.0
    - 3.1416
  resolution:
    - 41
    - 41
    - 41
  periodic_dims: [2]

control_space:
  n_dims: 2
  lo:
    - -1.0
    - 0.0
  hi: 
    - 1.0
    - 0.2

disturbance_space:
  n_dims: 0
  lo: None
  hi: None
